usecases/transfer_check_usecase.go

```diff
@@ -515,9 +513,7 @@ func (usecase *TransferCheckUsecase) ScoreTransfer(
 		},
 	)
 	if err != nil {
-		return models.Transfer{}, errors.Wrapf(
```

Comment:
The idea of this was that I don't want to leak any special errors coming from the decision usecase that could be interpreted as a distinct type of HTTP status in present_error to the transfercheck usecase.
However, I realize that context canceled errors are a counter-example for this.